addons/mail/static/src/new/core/thread_model.js

```diff
@@ -49,7 +49,7 @@ export class Thread {
             this[key] = data[key];
         }
         if (!this.composer) {
```

Comment:
it feels like this should only be done in "create" case and never changed again

Reply:
should I put it in the constructor then?

Reply:
constructor or next to "new" call

Reply:
changes done